Frontend/ui-driver/src/Screens/HomeScreen/Controller.purs

```diff
@@ -735,7 +745,13 @@ activeRideDetail state (RidesInfo ride) = {
   waitingTime : if (getValueToLocalStore IS_WAIT_TIMER_STOP) == "Stop" && state.props.timerRefresh then (getValueToLocalStore SET_WAITING_TIME) else state.data.activeRide.waitingTime,
   rideCreatedAt : ride.createdAt,
   waitTimeInfo : state.data.activeRide.waitTimeInfo,
-  requestedVehicleVariant : ride.requestedVehicleVariant
+  requestedVehicleVariant : ride.requestedVehicleVariant,
+  disabilityTag :  case ride.disabilityTag of
+              Just "BLIND_AND_LOW_VISION" -> Just ST.BLIND_AND_LOW_VISION
```

Comment:
Don't make data type for this disability. @kavya-shree-s mentioned in future they can add more types from backend so better maintain it as a string.

Reply:
Updated this case with 2 more conditions.
```
Just _ -> Just ST.OTHER_DISABILITY
Nothing -> Nothing
```